corehq/feature_previews.py

```diff
@@ -76,6 +76,14 @@ def preview_values_by_name(domain):
     help_link='https://help.commcarehq.org/display/commcarepublic/Adding+Icons+in+Case+List+and+Case+Detail+screen'
 )
 
+CONDITIONAL_ENUM = FeaturePreview(
+    slug='conditional_enum',
+    label=_('Conditional Enum in Case List'),
+    description=_(
+        "Conditional ID mapping in case tiles. "
```

Comment:
how about this: `Specify a custom xpath expression to calculate a lookup key in the case list or case detail screen enum column.`